core/src/main/java/greencity/exception/handler/CustomExceptionHandler.java

```diff
@@ -346,4 +343,20 @@ public final ResponseEntity<Object> handleBadRequestWhenProfilePictureExceeded(M
         log.error("Error when profile picture was being uploaded {}", me);
         return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(me.getMessage());
     }
+
+    /**
+     * Method intercept exception {@link IllegalArgumentException}.
+     *
+     * @param ex      Exception witch should be intercepted.
+     * @param request contain detail about occur exception
+     * @return ResponseEntity witch contain http status and body with message of
+     *         exception.
+     * @author Volodymyr Mladonov
+     */
+    @ExceptionHandler({IllegalArgumentException.class})
+    public ResponseEntity<Object> handleIllegalArgumentException(IllegalArgumentException ex, WebRequest request) {
+        log.info(ex.getMessage());
+        ExceptionResponse exceptionResponse = new ExceptionResponse(getErrorAttributes(request));
+        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(exceptionResponse);
+    }
```

Comment:
This is not a proper fix, since you've changed the logic in the entire application, instead of this please validate the field!